scrapyd/website.py

```diff
@@ -105,43 +105,124 @@ def __init__(self, root, local_items):
         self.local_items = local_items
 
     def render(self, txrequest):
-        cols = 8
-        s = "<html><head><title>Scrapyd</title></head>"
+        cols = 0
+
+        s = "<html><head><title>Scrapyd</title>"
+        s += """
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
The scrapyd website is meant to be minimal.
Using javascript libraries if far from this.
This feature can be implemented with plain forms.
We can even change the webservice to allow get requests
and make it as simple as a link next to each job.